### ISSUE TITLE
Extra selector calls

### ISSUE THREAD

**Issue by danielzagumennyi** [zustandjs/derive-zustand#2]
Hi!

Why does the selector call four times on the first render? (see console)
Can this be avoided?

https://codesandbox.io/s/vigorous-microservice-jnf92q

**Comment by dai-shi** [comment]
Yeah, it should probably be avoided. It's not optimized as of now.
Our project status is still to understand the feasibility of the API.

**Comment by dai-shi** [comment]
Let's leave it open meanwhile.
